coldfront/core/utils/validate.py

```diff
@@ -38,4 +41,19 @@ def validate_date(self):
             datetime.datetime.strptime(self.value.strip(), "%Y-%m-%d")
         except:
             raise ValidationError(
-                f'Invalid Value {self.value}. Date must be in format YYYY-MM-DD and date must be today or later.')
\ No newline at end of file
+                f'Invalid Value {self.value}. Date must be in format YYYY-MM-DD and date must be today or later.')
+    
+    def validate_json(self):
+        try:
+            _ = json.loads(self.value)
+        except:
+            raise ValidationError(
+                f'Invalid Value {self.value}. Value must be valid JSON.')
+    
+    def validate_billing_cycle(self):

```

Comment:
@peaches7083 the placement of this method was mentioned during the review meeting. I will comment once this is refactored.